data/json/overmap/overmap_special/specials.json

```diff
@@ -1366,7 +1366,7 @@
     "id": "Evac Shelter 1",
     "overmaps": [
       { "point": [ 0, 0, 0 ], "overmap": "shelter_1_north" },
-      { "point": [ 0, 0, -1 ], "overmap": "shelter_under_north" },
+      { "point": [ 0, 0, -1 ], "overmap": "shelter_under_1_north" },
```

Comment:
Can someone tell me if this is right? I'm getting a build failure because of this, but I'm following the pattern I see elsewhere in the file...

Reply:
You need to add `shelter_under_1` overmap terrain.